apps/src/p5lab/redux/animationPicker.js

```diff
@@ -229,9 +271,11 @@ export function pickNewAnimation() {
  * process. Dispatch root gamelab action to add appropriate metadata and then
  * close the animation picker.
  * @param {!AnimationProps} animation
+ * @param {!boolean} multiSelect - boolean determines if we are in the multiselect experiment. Will
+ *    be removed after experiment
  * @returns {function}
  */
-export function pickLibraryAnimation(animation) {
+export function pickLibraryAnimation(animation, multiSelect = false) {
```

Comment:
we shouldn't do this until multiSelect is fully-launched/default here, but `pickLibraryAnimation` doesn't isn't an accurate name in the context of multiSelect; we probably want to update it to something like `toggleLibraryAnimationSelected` or something that denotes it isn't purely additive anymore

Reply:
I'm definitely looking forward to when this is released and we can re-factor this whole function.